src/Compilers/Core/Portable/PEWriter/RootModuleType.cs

```diff
@@ -18,288 +23,163 @@ namespace Microsoft.Cci
     /// </summary>
     internal class RootModuleType : INamespaceTypeDefinition
```

Comment:
>RootModuleType [](start = 19, length = 14)

Please revert all refactorings in this file. Keep the changes to the minimum necessary to implement the feature.

Reply:
Sure. I made it easy to do so by keeping it an isolated commit.

Reply:
> Sure. I made it easy to do so by keeping it an isolated commit.

Feature work should not be mixed with unrelated refactorings. This significantly complicates review process and causes unnecessary merge conflicts when different features are merged. Also, refactorings in existing code based on stylistic preferences should be avoided in general. Different people prefer different styles, as long as there is no violation, the code is fine.

---
In reply to: [409482645](https://github.com/dotnet/roslyn/pull/43281#discussion_r409482645) [](ancestors = 409482645)